Core/source/mobile-interface/format-output.ts

```diff
@@ -114,7 +117,9 @@ export const fmtContentBlock = (allContentBlocks: MsgBlock[]): { contentBlock: M
     </head>
     <body>${msgContentAsHtml}</body>
   </html>`;
-  return { contentBlock: MsgBlock.fromContent('plainHtml', msgContentAsHtml), text: msgContentAsText.trim() };
+  const contentBlock = MsgBlock.fromContent('plainHtml', msgContentAsHtml);
+  contentBlock.verifyRes = verifyRes;
```

Comment:
The logic should probably be a bit more complicated, for safety.

 - if no input block was signed, then no signature on output block
 - else, if any of the input blocks contains an invalid signature, then that signature should be on the output block
 - else, if the input blocks were signed by a mixture of public keys, then pick the first one and add `mixed = true` to verifyRes (need to update type)
 - else, (at this point all available signatures are valid and from the same key) if only some input blocks were signed, add `partial = true` to verifyRes (need to update type)
 - else (at this point all are signed successfully by the same key, and no input blocks are missing signatures) pick the first verifyRes from input to pass to output